README.md

```diff
@@ -23,7 +23,15 @@ Thanks to the internal optimization, TiKV and TiDB can work together to be the b
 
 ### Required Rust version
 
-Rust Nightly is required.
+Rust Nightly is required. TiKV is currently tested mainly with `rust-nightly-2018-01-09`, however we would like to track `nightly`, so please report new breakage.
+
+```bash
+# Get rustup from rustup.rs, then:
+rustup install nightly
```

Comment:
do we need to assign a version here?

Reply:
If we're trying to track nightly in development, then no. If we'd like development to lock to a version and update it in jumps then we do. I'm not sure what we'd prefer.

Reply:
I think we should use the specified version here. We meet many times that the user uses a newer Rust version and fails to compile TiKV.